static/js/components/search/ModifiedSelectedFilter.js

```diff
@@ -13,6 +15,12 @@ export default class ModifiedSelectedFilter extends React.Component {
   }
 
   countryNameTranslations: Object = makeCountryNameTranslations()
+  countryNameList: string = _.values(this.countryNameTranslations)
+
+  isResidence = (labelKey: string) =>
```

Comment:
Can you change this name to `isLocation`? `"Residence"` only covers one of the two things we're looking for